packages/@sanity/form-builder/src/inputs/BlockEditor-slate/FormBuilderBlock.js

```diff
@@ -356,7 +392,32 @@ export default class FormBuilderBlock extends React.Component {
           ref={this.refPreview}
           className={styles.previewContainer}
         >
-          {this.renderPreview()}
+          {/* <div className={styles.dragHandle}>
+            <DragBarsIcon />
+          </div> */}
+          <div className={styles.preview}>
+            {this.renderPreview()}
+          </div>
+          <div className={styles.functions}>
+            {
+              memberType && <span className={styles.type}>{memberType.title || memberType.name}</span>
+            }
+            <div>
+              <Button
+                kind="simple"
+                icon={EditIcon}
+                title="Delete"
+              />
+            </div>
+            {/* <div>
```

Comment:
Add a todo note to the comment here too?

Reply:
👍